_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -265,6 +280,10 @@ class HealthCareApp extends React.Component {
           data: _.get(this.state.applicationData, statePath),
           onStateChange: (subfield, update) => {
             this.publishStateChange(statePath.concat(subfield), update);
+          },
+          app: {
```

Comment:
cute. :)

I think you can get away with just dumping `sectionCondition` into the proper part of `applicationData`.